lib/dgram.js

```diff
@@ -639,6 +639,25 @@ Socket.prototype.unref = function() {
   return this;
 };
 
+
+Socket.prototype.setRecvBufferSize = function(size) {
+  var err = this._handle.setRecvBufferSize(size);
+
+  if (err) {
+    throw errnoException(err, 'setRecvBufferSize');
+  }
+};
+
+
+Socket.prototype.setSendBufferSize = function(size) {
+  var err = this._handle.setSendBufferSize(size);
```

Comment:
Wow, completely forgot about the existence of `>>>`
So much fun working with talented people 🕺